backend/src/utils/database.ts

```diff
@@ -1,44 +1,339 @@
+  '42P04': 'duplicate_database',
+  '42723': 'duplicate_function',
+  '42P05': 'duplicate_prepared_statement',
+  '42P06': 'duplicate_schema',
+  '42P07': 'duplicate_table',
+  '42712': 'duplicate_alias',
+  '42710': 'duplicate_object',
+  '42702': 'ambiguous_column',
+  '42725': 'ambiguous_function',
+  '42P08': 'ambiguous_parameter',
+  '42P09': 'ambiguous_alias',
+  '42P10': 'invalid_column_reference',
+  '42611': 'invalid_column_definition',
+  '42P11': 'invalid_cursor_definition',
+  '42P12': 'invalid_database_definition',
+  '42P13': 'invalid_function_definition',
+  '42P14': 'invalid_prepared_statement_definition',
+  '42P15': 'invalid_schema_definition',
+  '42P16': 'invalid_table_definition',
+  '42P17': 'invalid_object_definition',
+  '44000': 'with_check_option_violation',
+  '53000': 'insufficient_resources',
+  '53100': 'disk_full',
+  '53200': 'out_of_memory',
+  '53300': 'too_many_connections',
+  '53400': 'configuration_limit_exceeded',
+  '54000': 'program_limit_exceeded',
+  '54001': 'statement_too_complex',
+  '54011': 'too_many_columns',
+  '54023': 'too_many_arguments',
+  '55000': 'object_not_in_prerequisite_state',
+  '55006': 'object_in_use',
+  '55P02': 'cant_change_runtime_param',
+  '55P03': 'lock_not_available',
+  '55P04': 'unsafe_new_enum_value_usage',
+  '57000': 'operator_intervention',
+  '57014': 'query_canceled',
+  '57P01': 'admin_shutdown',
+  '57P02': 'crash_shutdown',
+  '57P03': 'cannot_connect_now',
+  '57P04': 'database_dropped',
+  '58000': 'system_error',
+  '58030': 'io_error',
+  '58P01': 'undefined_file',
+  '58P02': 'duplicate_file',
+  '72000': 'snapshot_too_old',
+  F0000: 'config_file_error',
+  F0001: 'lock_file_exists',
+  HV000: 'fdw_error',
+  HV005: 'fdw_column_name_not_found',
+  HV002: 'fdw_dynamic_parameter_value_needed',
+  HV010: 'fdw_function_sequence_error',
+  HV021: 'fdw_inconsistent_descriptor_information',
+  HV024: 'fdw_invalid_attribute_value',
+  HV007: 'fdw_invalid_column_name',
+  HV008: 'fdw_invalid_column_number',
+  HV004: 'fdw_invalid_data_type',
+  HV006: 'fdw_invalid_data_type_descriptors',
+  HV091: 'fdw_invalid_descriptor_field_identifier',
+  HV00B: 'fdw_invalid_handle',
+  HV00C: 'fdw_invalid_option_index',
+  HV00D: 'fdw_invalid_option_name',
+  HV090: 'fdw_invalid_string_length_or_buffer_length',
+  HV00A: 'fdw_invalid_string_format',
+  HV009: 'fdw_invalid_use_of_null_pointer',
+  HV014: 'fdw_too_many_handles',
+  HV001: 'fdw_out_of_memory',
+  HV00P: 'fdw_no_schemas',
+  HV00J: 'fdw_option_name_not_found',
+  HV00K: 'fdw_reply_handle',
+  HV00Q: 'fdw_schema_not_found',
+  HV00R: 'fdw_table_not_found',
+  HV00L: 'fdw_unable_to_create_execution',
+  HV00M: 'fdw_unable_to_create_reply',
+  HV00N: 'fdw_unable_to_establish_connection',
+  P0000: 'plpgsql_error',
+  P0001: 'raise_exception',
+  P0002: 'no_data_found',
+  P0003: 'too_many_rows',
+  P0004: 'assert_failure',
+  XX000: 'internal_error',
+  XX001: 'data_corrupted',
+  XX002: 'index_corrupted',
 };
```

Comment:
I'll move it to a file, yh, those errors will never change, those are the errors built into postgresql, so aslong as we use the same version of Postgres this will never change.